docs/sql-programming-guide.md

```diff
@@ -1890,10 +1890,10 @@ that these options will be deprecated in future release as more optimizations ar
   </tr>
   <tr>
     <td><code>spark.sql.codegen</code></td>
-    <td>false</td>
+    <td>true</td>
     <td>
       When true, code will be dynamically generated at runtime for expression evaluation in a specific
-      query.  For some queries with complicated expression this option can lead to significant speed-ups.
+      query. For some queries with complicated expression this option can lead to significant speed-ups.
       However, for simple queries this can actually slow down query execution.
```

Comment:
@davies which one is not needed -- 'this option can lead to significant speed-ups' or 'for simple queries this can actually slow down query execution'. Either way can you update it @KaiXinXiaoLei ?